test/extended/apiserver/latency.go

```diff
@@ -0,0 +1,85 @@
+package apiserver
+
+import (
+	"context"
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	promhelper "github.com/openshift/origin/test/extended/util/prometheus"
+)
+
+var (
+	oc                   *exutil.CLI
+	url, bearerToken, ns string
+	execPod              *v1.Pod
+)
```

Comment:
These will either be removed or moved to a function once https://github.com/openshift/origin/pull/26740 is merged. I just exposed them in the meantime to avoid changing the signature of `expectRequestLatency` when the PR will be merged.